### DISCUSSION TITLE
Dropping Windows kanidmd Support

### DISCUSSION THREAD

**Discussion by yaleman** [kanidm/kanidm#1906] [Polls]
We're considering dropping support for `kanidmd` running on Windows as complexity of the daemon continues to increase.

It's runnable as a container for folks who base their infrastructure on Windows, and the client tools will stay supported, we're just looking to stop supporting the daemon until someone offers reasons to keep it, and also support for its lifecycle.

This poll is going to stay open for at least a month, at which time the decision will be noted and acted upon.

**Comment by yaleman** [comment]
Done! Dropping Windows builds. 😄

**Comment by trissylegs** [comment]
A bit late to this but as the original requester of Windows: I no longer have a use for it (escaped windows land).
